README.md

```diff
@@ -45,57 +41,63 @@ The JFrog VS Code Extension adds JFrog Xray scanning of project dependencies to
 
-<a name="need-a-free-jfrog-instance"></a>
-### Need a FREE JFrog instance?
-Don't have a running JFrog instance? No problem! [Get a FREE instance in the cloud](https://jfrog.com/start-free/).
+## Getting Started
+1. Install the **JFrog** extension in VS Code
+2. Need a FREE JFrog environment in the cloud? [Create one now and connect VS Code to it](#set-up-a-free-jfrog-environment-in-the-cloud)
+3. Already have a working JFrog environment? [Connect VS Code to it](#connecting-vs-code-to-your-jfrog-environment)
+4. [Start](#using-the-extension) using the extension
 
-### Component Tree Icons
-The icon demonstrates the top severity issue of a selected component and its transitive dependencies. The following table describes the severities from lowest to highest:
+## Set Up a FREE JFrog Environment in the Cloud
+Need a FREE JFrog environment in the cloud, so that VS Code can connect to it? Just run one of the following commands in your terminal. The commands will do the following.
 
-|                 Icon                | Severity |                                       Description                                      |
-|:-----------------------------------:|:--------:|:---------------------------------------------------------------------------------------|
-|   ![Normal](resources/normal.png)   |  Normal  | Scanned - No Issues                                                                    |
-|  ![Unknown](resources/unknown.png)  |  Unknown | No CVEs attached to the vulnerability or the selected component not identified in Xray |
-|      ![Low](resources/low.png)      |    Low   | Top issue with low severity                                                            |
-|   ![Medium](resources/medium.png)   |  Medium  | Top issue with medium severity                                                         |
-|     ![High](resources/high.png)     |   High   | Top issue with high severity                                                           |
-| ![Critical](resources/critical.png) | Critical | Top issue with critical severity                                                       |
+1. Install JFrog CLI on your machine.
+2. Create a FREE JFrog environment in the cloud for you.
+3. Configure VS Code to connect to your new environment.
+
+**MacOS and Linux using cUrl**
+```
+curl -fL https://getcli.jfrog.io/setup | sh
+```
 
-## General Configuration
-### Configuring JFrog Platform
-Connect to JFrog Platform by clicking on the green Connect ![Connect](resources/readme/connect.png) button:
+**Windows using PowerShell**
+```
+powershell "Start-Process -Wait -Verb RunAs powershell '-NoProfile iwr https://releases.jfrog.io/artifactory/jfrog-cli/v2-jf/[RELEASE]/jfrog-cli-windows-amd64/jf.exe -OutFile $env:SYSTEMROOT\system32\jf.exe'" && jf setup
+```
+
+## Connecting VS Code to Your JFrog Environment
```

Comment:
Should we add a comment that the extension will connect automatically to a configured server in the JFrog CLI?

Reply:
I'm afraid this will be a bit confusing. I suggest we skip this one.